gateway-messages/src/sp_to_mgs.rs

```diff
@@ -944,6 +953,24 @@ pub struct UpdateInProgressStatus {
     pub total_size: u32,
 }
 
+/// Represents the result of a successful [`SetPowerState`] request.
+/// [`SetPowerState`]: crate::MgsRequest::SetPowerState
+#[derive(Copy, Clone, PartialEq, Eq)]
+#[repr(u8)]
+pub enum PowerStateTransition {
+    Changed = 1,
```

Comment:
Channeling my inner @mkeeter - do we need to skip `0`?

Reply:
You probably don't need the `repr(u8)` either 😄